### ISSUE TITLE
Unity WebGL support

### ISSUE THREAD

**Issue by Honk5000** [sungaila/PDFtoImage#105]
### Question

Hi, I wanted to use PDFToImage to view Pdfs in Unity. That works great but when I build for WebGL I get "DLLNotFoundException: unable to load DLL 'pdfium'".
Am I missing something? Would it be possible to add WebGL support?
I saw that there are pdfium builds for WASM but I am not sure how I could use them in combination with PDFToImage.

**Comment by sungaila** [comment]
Hi @Honk5000,

thanks for your question! Unfortunately, I don't know much about Unity web deployment. We would need two native libraries for this to work: SkiaSharp and pdfium.

What would be needed for the web build? Maybe a wasm file? If you have knowledge about that, please let me know. Right now I am searching the Unity documentation for any hints on how to include native libraries.

**Comment by Honk5000** [comment]
Hi @sungaila,
thank you for your fast reply. It is unfortunate that Unity has these weird version requirements for Emscripten. I tried all day to find a solution but in the end I got another asset from the Unity Asset Store that could rasterise pdfs to images: [pdf renderer](https://assetstore.unity.com/packages/tools/gui/pdf-renderer-32815)
They also use pdfium and it looks like they use a JavaScript implementation of the library when using WebGL.

Thank you again for your help and time
